src/app/app-routing.module.ts

```diff
@@ -17,6 +17,6 @@ export const routes: Routes = [
 @NgModule({
   imports: [RouterModule.forRoot(routes)],
   exports: [RouterModule],
-  providers: [{ provide: APP_BASE_HREF, useValue: "/periodo/" }],
+  providers: [{ provide: APP_BASE_HREF, useValue: "/periodos/" }],
```

Comment:
```suggestion
  providers: [{ provide: APP_BASE_HREF, useValue: "/periodo/" }],
```
aquí creo se debe **periodo** ya que la url del menú está así actualmente: 
`https://pruebassgamicroclientes.portaloas.udistrital.edu.co/periodo/list-periodo`